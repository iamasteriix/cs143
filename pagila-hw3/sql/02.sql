/*
 * Compute the country with the most customers in it. 
 */

SELECT country
FROM country
JOIN city USING (country_id)
JOIN address USING (city_id)
JOIN customer USING (address_id)
GROUP BY country.country
HAVING count(customer_id) = (
    SELECT max(count_customers)
    FROM (
        SELECT count(customer_id) AS count_customers
        FROM customer
        JOIN address USING (address_id)
        JOIN city USING (city_id)
        JOIN country USING (country_id)
        GROUP BY country.country) t
);
