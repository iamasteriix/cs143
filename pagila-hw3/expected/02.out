 country 
---------
 India
(1 row)

